spec/index.bs

```diff
@@ -5204,7 +5204,7 @@ Note that member uses several binding slots, as defined there.
     - Metadata for converting coordinates before reading from those planes (crop and rotation).
     - Metadata for converting values into the specified output color space (matrices, gammas, 3D LUT).
 
-    The configuration used internally by an implementation may not be consistent across time,
+    The configuration used internally by an implementation may be inconsistent across time,
```

Comment:
Since you're replacing may everywhere else, might as well here too?

```suggestion
    The configuration used internally by an implementation might be inconsistent across time,
```